official/utils/arg_parsers/parsers.py

```diff
@@ -202,6 +204,33 @@ def __init__(self, add_help=False, num_parallel_calls=True, inter_op=True,
       )
 
 
+class DistributionStrategiesParser(argparse.ArgumentParser):
+  """Parser to add flags need for distribution strategies.
+
+  Args:
+    add_help: Create the "--help" flag. False if class instance is a parent.
+    batch_size: Create a flag to specify the batch size. (Instead of the one
+      in BaseParser)
+  """
+
+  def __init__(self, add_help=False):
+    super(DistributionStrategiesParser, self).__init__(add_help=add_help)
+
+    self.add_argument(
+        "--use_distribution_strategy", "-uds",
+        action="store_true",
+        help="If set, use the DistributionStrategies API."
+    )
+
+    self.add_argument(
+        "--gpus_for_distribution_strategy", "-gds",
+        type=int, default=2,
+        help="[default: %(default)s] How many GPUs to use with the "
+             "DistributionStrategies API.",
```

Comment:
We need links to the docs somewhere, and this seems like a good place. Maybe also in the comment noting that multi-GPU is experimental.

Reply:
we have a reasonable README here that we can link to? 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/distribute/README.md